source/common/upstream/health_discovery_service.cc

```diff
@@ -40,8 +46,7 @@ void HdsDelegate::sendHealthCheckRequest() {
 }
 
 void HdsDelegate::handleFailure() {
-  ENVOY_LOG(warn, "Load reporter stats stream/connection failure, will retry in {} ms.",
-            RETRY_DELAY_MS);
+  ENVOY_LOG(warn, "HdsDelegate stream/connection failure, will retry in {} ms.", RETRY_DELAY_MS);
```

Comment:
Can you add a TODO to use jittered backoff as in https://github.com/envoyproxy/envoy/pull/3791 at some point in the future?

Reply:
Done